### DISCUSSION TITLE
GitHub Marketplace doesn't show open-source Actions when filtering for "Free" category

### DISCUSSION THREAD

**Discussion by myyk** [community/community#26899] [Actions]
<p>Hi,</p>
<p>I released a plugin which you can find in GitHub Marketplace under this search: <a href="https://github.com/marketplace?query=git-democracy" rel="noopener nofollow ugc">https://github.com/marketplace?query=git-democracy</a></p>
<p>If you add the free category to the filters, it is filtered out:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/marketplace?category=free&amp;query=git-democracy" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/361;"><img src="https://user-images.githubusercontent.com/2080820/181100526-94b7331e-fd9e-4b2d-a1a0-2ddec18e8d09.png" class="thumbnail" width="690" height="361" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/3395134196dc5da768b0f22edf0b417989971002_2_690x361.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/3395134196dc5da768b0f22edf0b417989971002_2_1035x541.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/3/3395134196dc5da768b0f22edf0b417989971002.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/3395134196dc5da768b0f22edf0b417989971002_2_10x10.png"></div>

<h3><a href="https://github.com/marketplace?category=free&amp;query=git-democracy" target="_blank" rel="noopener nofollow ugc">GitHub Marketplace: tools to improve your workflow</a></h3>

<p>Find the tools that help your team build better, together.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Yet my action is and should be free to use since it’s completely open sourced under an MIT license.</p>
<p>I don’t see anything in the action metadata to make the plugin free. Am I missing something or is this a bug?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/myyk">@myyk</a>,</p>
<p>The ‘free’ category is only applied to <code>Github Apps</code>, not for <code>Github Actions</code>.</p>
<p>For Github action, it will caculate action minutes for charge, you can refer to billing doc <a href="https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions" rel="noopener nofollow ugc">here</a> for more details.</p>
<p>Thanks</p>
